nbs/01_Tutorials/01_tutorial.ipynb

```diff
@@ -432,67 +432,32 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "nbdev hosts your docs on GitHub Pages---an excellent (and free!) way to host websites."
+    "nbdev hosts your docs on GitHub Pages---an excellent (and free!) way to host websites. nbdev automatically enables GitHub Pages when you create a nbdev-enabled GitHub repo."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You can enable it for your repo by clicking on the \"Settings\" tab near the top-right of your repo page, then \"Pages\" on the left, then setting the \"Branch\" to \"gh-pages\", and finally clicking \"Save\"."
+    "By default, your docs should be available at: `https://{user}.github.io/{repo}`. For example, you can view `fastai`'s `nbdev-hello-world` docs at <https://fastai.github.io/nbdev-hello-world>:"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "::: {.callout-note}\n",
-    "\n",
-    "nbdev uses GitHub Pages by default because its easily accessible, however you can use any host you like.\n",
-    "\n",
-    ":::"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "It should look similar to this after you click \"Save\":"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "![](images/github-enable-pages.png){.pb-2 fig-align=\"center\" .border .rounded .shadow-sm}"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Head back to GitHub Actions and you should see a new workflow run: \"pages build and deployment\". As the name says, this workflow deploys your website contents to GitHub Pages."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "![](images/github-actions-pages.png){fig-align=\"center\" .border .rounded .shadow-sm}"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Wait for the workflow run to complete, then open your website. By default it should be available at: `https://{user}.github.io/{repo}`. For example, you can view `fastai`'s `nbdev-hello-world` docs at <https://fastai.github.io/nbdev-hello-world>."
+    "![](images/nbdev-hello-world-site-initial.png){fig-align=\"center\" .border .rounded .shadow-sm}"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "![](images/nbdev-hello-world-site-initial.png){fig-align=\"center\" .border .rounded .shadow-sm}"
+    "::: {.callout-note}\n",
+    "\n",
+    "nbdev uses GitHub Pages by default because its easily accessible. However, you can use any host you like.  The files for the static site are located in a local `/_docs` directory at the root of your repository after running the command `nbdev_docs` .  This directory is not checked into git and is ignored by `.gitignore`, but you can use these files to deploy to any platform you want.\n",
```

Comment:
I think the part that describes `_docs` is also TMI. I'm going to move that to the explainer section as well in a follow on PR